src/components/organisms/Sider/Sider.tsx

```diff
@@ -95,6 +96,16 @@ const getRoutes = (showSocialLinksInSider: boolean) => [
           active: /environment-management/,
         },
       ]),
+
+  {
```

Comment:
Will it be available in OSS?

Reply:
no... but we most likely want to tease it on OSS :)

Reply:
We should have some page then on this `/status-page` URL, or disable this button when it's not available 👍 Probably it would be best to do it with plugins